docs/annotators.md

```diff
@@ -194,16 +194,16 @@ comments: true
 === "Mask"
 
     ```python
-    >>> import supervision as sv
+    import supervision as sv
 
-    >>> image = ...
-    >>> detections = sv.Detections(...)
+    image =
```

Comment:
Type like 'image = ...'   or place image file name or load via image cv2 for making more readable. Same for others